scripts/generator/graphql_generator/csharp/SDK/GlobalGameObject.cs.erb

```diff
@@ -5,6 +5,16 @@ namespace <%= namespace %>.SDK {
     public class GlobalGameObject {
         private static GameObject GameObject;
 
+        public static string GameObjectName {
```

Comment:
Nit. Maybe this should be simply called `name` since `AddComponent` is actually a method of a `GameObject`.